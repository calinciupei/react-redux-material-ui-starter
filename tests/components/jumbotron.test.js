import React from 'react';
import { shallow } from 'enzyme';

import Jumbotron from '../../src/components/jumbotron/jumbotron.component';

let context;
let props = {
  classes: {
    jumbotron: 'Jumbotron-jumbotron-1',
    mainText: 'Jumbotron-mainText-3',
    textField: 'Jumbotron-textField-2'
  }
};

describe('<Jumbotron />', () => {
  context = shallow(<Jumbotron {...props} />);

  it('should render correctly', () => {
    expect(context.dive()).toMatchSnapshot();
  });
});
