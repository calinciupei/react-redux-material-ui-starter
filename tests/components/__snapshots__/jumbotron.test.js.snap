// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Jumbotron /> should render correctly 1`] = `
<WithStyles(Grid)
  alignItems="center"
  className="Jumbotron-jumbotron-1 Jumbotron-jumbotron-1"
  container={true}
  direction="row"
  justify="center"
>
  <WithStyles(Grid)
    item={true}
    xs={12}
  >
    Jumbotron
  </WithStyles(Grid)>
</WithStyles(Grid)>
`;
